plugins_examples/archive/archive.go

```diff
@@ -0,0 +1,50 @@
+package main
```

Comment:
i would name this plugin `archive_urls.go`  since it's only archiving urls :)

Reply:
done!